targets/nasm_cve-2018-16517/config.sh

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
It seems that the shebang must be on the first line. The output from shellcheck:
```
targets/nasm_cve-2018-16517/config.sh:2:1: error: The shebang must be on the first line. Delete blanks and move comments. [SC1128]
```